### PR TITLE
New search filter

### PR DESCRIPTION
**Pull request by RobertWDLowe**

### CODE REVIEW

**Comment by ollietreend** [comment]
@marcincichon **OR**

File: wp-content/themes/mojintranet/models/taxonomy.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    return $options;
+  }
+
+  private function _format_row($term) {
+    return [
+      'id' => $term->term_id,
+      'name' => $term->name,
+      'slug' => $term->slug,
+      'count' => $term->count,
```
Comment (RobertWDLowe) [inline wp-content/themes/mojintranet/models/taxonomy.php]: Get rid of count

**Comment by marcincichon** [comment]
@ollietreend wasn't OR after all but fixed now and available on dev2.

**Comment by RobertWDLowe** [comment]
@marcincichon  looks good.

**Comment by ollietreend** [comment]
Looks good @marcincichon 👍

**Pull request merged**